package.json

```diff
@@ -59,6 +59,6 @@
   "devDependencies": {
     "babel-plugin-styled-components": "^1.5.1",
     "babel-plugin-transform-object-assign": "^6.22.0",
-    "prettier": "^1.17.0"
+    "prettier": "~1.17.0"
```

Comment:
Changed per https://github.com/iterative/dvc.org/issues/363#issuecomment-494557734

Reply:
if this change fixes that ticket, please add it to the PR description - like Fixes #... so that we don't forget to close it. Thanks :)

Reply:
It doesn't completely fix it, these changes only address the first checkbox from that issue.